docs/_pages/grammar.md

```diff
@@ -92,6 +93,6 @@ TableIndexer = '[' Type ']' ':' Type
 TableProp = NAME ':' Type
 TablePropOrIndexer = TableProp | TableIndexer
 PropList = TablePropOrIndexer {fieldsep TablePropOrIndexer} [fieldsep]
-TableType = '{' [PropList] '}'
```

Comment:
I don't think this is quite right, this should be `'{' Type '}' | '{' [PropList] '}'`

Reply:
sgtm